Dockerfile

```diff
@@ -1,6 +1,8 @@
 # Build the manager binary
 FROM golang:1.16 as builder
 WORKDIR /workspace
+
+ENV GOPROXY=direct
```

Comment:
Do we want this here? I was under the impression that this is solely for dev desktop

Reply:
This should be fine as it should work for everyone across the board. I don't see any cons of having this set.